docs/advanced-configuration.md

```diff
@@ -883,6 +883,22 @@ agent:
            storage: null
 ```
 
+## Using edge processor with OTel
+
+The [Edge Processor solution](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-ebpf) is a data processing tool that can be used to enrich and filter your logs using SPL2.
+
+Edge processor can be used to send data to various destinations, such as Splunk cloud or an Amazon S3 bucket. You can read more about [destinations here](https://help.splunk.com/en/splunk-cloud-platform/process-data-at-the-edge/use-edge-processors/9.3.2411/send-data-out-from-edge-processors/how-the-destination-for-edge-processor-works).
+
+To configure a basic HEC destination you can use a basic configuration like this:
```

Comment:
```suggestion
To configure a HEC Edge processor destination you can use a basic configuration like this:
```
I assume this section is meant to instruct a user how to send data to the edge processor. I think if we could clarify this somewhere even before this config it would be helpful.